### PR TITLE
*: add cycle.yml

### PR DESCRIPTION
**Pull request by coocood**
Fix CI

### CODE REVIEW

File: pkg/util/codec_test.go [hunk 1]
```diff
@@ -2,31 +2,33 @@ package util
 
 import (
 	"bytes"
+	"testing"
 
-	"github.com/golang/protobuf/proto"
-	. "github.com/pingcap/check"
```
Comment (shenli) [inline pkg/util/codec_test.go]: Why not use `check` package?

Comment (coocood) [inline pkg/util/codec_test.go]: Adds more dependency.

Comment (coocood) [inline pkg/util/codec_test.go]: We don't have this package in vendor.

**Comment by shenli** [comment]
LGTM

**Pull request merged**